workers/unity/Packages/com.improbable.gdk.core/Utility/EntityManagerExtensions.cs

```diff
@@ -8,7 +8,7 @@ namespace Improbable.Gdk.Core
     // Attempt to validate the method as soon as possible when in the editor.
     [UnityEditor.InitializeOnLoad]
 #endif
-    internal static class EntityManagerExtensions
+    public static class EntityManagerExtensions
```

Comment:
Keep in mind this exposes the extension methods to all users of the core

Reply:
yeah... to be honest I don't like the extension method in general, as it is confusing even if used only internally...

will add it to my list of things to clean up